StandardConfig/production/ILDReconstruction.py

```diff
@@ -47,6 +48,12 @@
     "ILD_s5_o3_v02",
     "ILD_s5_o4_v02",
 )
+# only FCCMDI
+FCCeeMDI_DETECTOR_MODELS = (
+    "ILD_l5_o1_v09",
+    "ILD_l5_v11",
+)
+DETECTOR_MODELS = DETECTOR_MODELS + FCCeeMDI_DETECTOR_MODELS
```

Comment:
```suggestion
```
This has the potential for subtle issues, because the lists / sets are no longer disjoint, which can make some checks a bit confusing.

We can just do this in the argparse call below
```python
parser.add_argument(
    "--detectorModel",
    help="Which detector model to run reconstruction for",
    choices=DETECTOR_MODELS + FCCeeMDI_DETECTOR_MODELS,
    type=str,
    default="ILD_l5_o1_v02",
)
```